### ISSUE TITLE
Request: Add variable for xy coordinates of industries

### ISSUE THREAD

**Issue by 2TallTyler** [OpenTTD/nml#218]
Industries and industrytiles currently have no way of knowing their location on the map, unlike houses which have `x_coordinate` and `y_coordinate` variables. I would like to request similar variables for industries.

My desired use case is generating Import/Export industries only near the edge of the map. Another case would be separating the map into quadrant "regions" with different primary resources (coal, iron, etc) available in each.

On Discord, @andythenorth informs me that "80+ var 00" contains the xy coordinates of the industry's northernmost tile, per [this page](http://marcin.ttdpatch.net/sv1codec/TTD-locations.html#_IndustryArray).

I am not knowledgeable enough to know how to use this, and the [depreciated syntax](https://newgrf-specs.tt-wiki.net/wiki/NML:Deprecated_syntax) page of the docs suggested opening a feature request to formally implement this in NML.

**Comment by frosch123** [comment]
General warning: The linked page is TTDPatch specific, and not everything on it applies to OpenTTD.
In this case the description of var 80+x assumes a 256x256 map :p

For testing you can use this:
```
X coordinate = var[0x80, 0, 0xFFFFFFFF] % map_x_edge
Y coordinate = var[0x80, 0, 0xFFFFFFFF] / map_x_edge
```

**Comment by 2TallTyler** [comment]
Thank you for pointing me in the right direction. The test code you gave me works great!

**Comment by andythenorth** [comment]
I have case where I need a UUID for industry instances.  I am not aware of an actual UUID that is grf-readable (maybe in 80+?).  

xy is unique, so if there was a way to parse the returned value for uniqueness, it would work for my case.

**Comment by andythenorth** [comment]
This issue could be closed IMHO :)

**Comment by 2TallTyler** [comment]
I disagree. It would be nice to have an easy variable to read (and put in the docs) rather than forcing the user to perform the above calculation, even if it does work.
